### DISCUSSION TITLE
Inspector should not spam Output with useless messages

### DISCUSSION THREAD

**Discussion by miv391** [godotengine/godot-proposals#7445] [Editor]
When you change anything in Inspector, it spams useless messages. For example, when changing any color, you can get hundreds of messages. This feels like a debug feature that is just forgotten. There could be a flag in the editor settings where you could enable this feature if it is really needed, but by default this spamming should be disabled.

![kuva](https://github.com/godotengine/godot-proposals/assets/49998025/4e33e72c-9e97-474b-ae23-a2f5a15f6ea5)

**Comment by Calinou** [comment]
Now that the History dock exists, I think these messages could be removed from the Output panel entirely.

Note that you can also enable message collapsing on the right of the Output panel.

@KoBeWi What do you think?

**Reply by KoBeWi** [reply]
Output action logs provide a bit more information than History dock. You know when something happens, because output is usually visible, unlike History dock. Also when you undo, the log provides context where did the undo happen (scene/global/remote); the History dock does not show remote history.

That said, I didn't see console spam in any other software, even if it didn't have proper history navigation. At most such messages are displayed on some status bar. Maybe we could limit the prints to verbose mode?

**Reply by Calinou** [reply]
> Maybe we could limit the prints to verbose mode

The messages can already be hidden this way:

![image](https://github.com/godotengine/godot-proposals/assets/180032/9f40ebc2-11ed-408a-8ad9-8fefd5478875)

**Reply by KoBeWi** [reply]
But it disables more than undo/redo log. I meant that only the undo/redo messages could be limited to verbose mode, because they are more verbose than practically needed.
